Makefile

```diff
@@ -61,7 +62,10 @@ translate-server:
 
 pypi:
 	make doc-man
-	python setup.py sdist upload
+	rm -fr dist
+	python setup.py sdist
+	gpg --detach-sign -a --default-key ${SIGNING_KEY} dist/*.tar.gz
+	twine upload dist/*.tar.gz dist/*.tar.gz.asc
```

Comment:
should we add twine to the `requirements(-devel).txt`? Or is this a "this runs only on my machine" issue?

Reply:
I like this kind of issue! :-)
Well, twine is only used if you want to upload the package to pypi. To upload you need an account there. So if 3rd persons would like to develop in this environment they definitively will not need twine.